### DISCUSSION TITLE
`native` blocks for multiple languages in only one Fusion source file?

### DISCUSSION THREAD

**Discussion by ghost** [pfusik/cito#111] [Q&A] [answered]
Is it possible? I don't want to have multiple `.fu` source files with only the `native` blocks that are different.

**Comment by pfusik** [comment, marked as answer]
Use [conditional compilation](https://github.com/fusionlanguage/fut/blob/master/doc/reference.md#conditional-compilation):

```c
#if JAVA
System.gc(); // Java code
#endif
```

    fut -D JAVA -o Foo.java foo.fu
